tqecjs/src/workspace/components/button.js

```diff
@@ -0,0 +1,50 @@
+import { Text, Container, Graphics } from 'pixi.js';
+import { DropShadowFilter } from '@pixi/filter-drop-shadow';
+
+export const button = (text, x, y, color = 0x000000) => {
+	// Create the button container
+	const button = new Container();
+	// Create the button text
+	const buttonText = new Text(text, {
+		fontFamily: 'Arial',
+		fontSize: 15,
```

Comment:
15 looks like it should be a constant (or variable, for button and label scaling)